cypress/integration/Page/navigationPage.js

```diff
@@ -0,0 +1,11 @@
+export class NavigationPage{
+    publicAuctionPage(){
+        cy.get('.uhf-user__locations').click()
+        cy.contains('U.S. Locations').click()
+        cy.contains('Manheim Cincinnati').click()
+        cy.contains('Public Auctions').click()
+        cy.contains('See Inventory').click()
+        cy.wait(10000)
```

Comment:
do not use hard sleep. try `cy.get('element on the next page').should('be.visible')`